### DISCUSSION TITLE
Downloading images from minio server running in docker

### DISCUSSION THREAD

**Discussion by yichong96** [minio/minio#13962] [Q&A]
I managed to download a minio image in a bucket using HTTP GET request by running the command `mc policy set download <server_alias>/<bucket>`. Then I run minio on local host via `minio server ~/minio`. I managed to download bucket images via the image_url  (example: `http://localhost:9000/<bucket>/image_1.jpg`.)

I then run minio on docker. This is the docker run command `docker run --rm -d -p 9000:9000 -p 9001:9001 --network user_network --name minio -"MINIO_ROOT_USER=user1" -e "MINIO_ROOT_PASSWORD=user1" v ~/minio:/data minio/minio server /data --console-address ":9001"` as per the documentation. I then tried to do a HTTP get request on another dockerised application in the same `user_network`. 

I get an HTTP error 
`requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: http://minio:9000/query/image_1.jpg` where `query` is the bucket in the localhost `~/minio` directory. 

**NOTE**
when I connect to minio on localhost:9001 I somehow do not have the buckets that I had originally in localhost, although the `/data/` directory in the container is populated with my original buckets. 

How should I download images in minio running on docker ?

Thank you for your help !

**Comment by harshavardhana** [comment]
> ocker run --rm -d -p 9000:9000 -p 9001:9001 --network user_network --name minio -"MINIO_ROOT_USER=user1" -e "MINIO_ROOT_PASSWORD=user1" v ~/minio:/data minio/minio server /data --console-address ":9001"

You are missing a `-e` and a `-v` 

> docker run --rm -d -p 9000:9000 -p 9001:9001 --network user_network --name minio -e "MINIO_ROOT_USER=user1" -e "MINIO_ROOT_PASSWORD=user1"  -v ${HOME}/minio:/data minio/minio:RELEASE.2021-12-20T22-07-16Z server /data --console-address ":9001"

**Reply by yichong96** [reply]
oops sorry I copy pasted wrongly.. This is the actual command `docker run --rm -d -p 9000:9000 -p 9001:9001 --network user_network -e "MINIO_ROOT_USER=user_1" -e "MINIO_ROOT_PASSWORD=user1_password" --name minio -v ~/minio:/data minio/minio server /data --console-address ":9001"`

somehow the minio console does not let me see preexisting data when I connect to `localhost:9001 ` even though I have mounted `~/minio` to `/data `and `/data` in the container contains my buckets.

Is there a reason why the minio console looks like this with the create bucket button disabled ? 
<img width="1440" alt="Screenshot 2021-12-22 at 8 31 31 PM" src="https://user-images.githubusercontent.com/32008333/147093408-0d60d0e5-9cbd-4065-9c9e-06bf337b9e59.png">
